build/ci/cloudbuild.website_testing.yaml

```diff
@@ -13,16 +13,38 @@
 # limitations under the License.
 
 steps:
+  # Run Website sanity
+  - id: website_adversarial
```

Comment:
I'm wondering if it's worthwhile keeping these cloudbuild files around so an individual test can be triggered manually?

Reply:
Now we can still trigger those, but they would all run, would that work? I am bit worried maintaining several copies of very similar files..